DOC-2372-updated env variable syntax (#1905)
harness/developer-hub: docs/platform/12_Variables-and-Expressions/harness-variables.md

For more information, go to [Services and Environments Overview](/docs/continuous-delivery/get-started/services-and-environments-overview).

To reference an environment-level variable, use the expression `<+env.variables.variableName>`.

> For example, here is an environment variable named `envvar`. You would reference it as `<+env.variables.envvar>`.

Reviewer: For example, to reference an environment variable `envvar`, use the following expression:
`<+env.variables.envvar>`

> For example, here is an environment variable named `envvar`. You would reference it as `<+env.variables.envvar>`.

Author: ```suggestion
For example, to reference an environment variable named `envvar`, use the following expression:

`<+env.variables.envvar>`
```
